TASK_ID=patch-aays1-panel-from-115-completion-20260706
PAGE_KEY=aays1
RUNNER_V4=20260706
work_root=C:\AAYS_WT\AAYS_STABLE_RUNNER_WORKTREES
node_exists=True
npm_exists=True
edge_or_chrome_exists=True
playwright_available=False
site_8010_ok=True
site_8020_ok=False
browser_smoke_passed=True
automation_exit_code=0
fake_data=false
--- output ---
PATCHED=4
